### ISSUE TITLE
Assure consistent Novelty Calculation

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#420]
The Novelty scores for given dungeons still seem to be subtly affected by how they are loaded. If you load a dungeon you can calculate its novelty repeatedly and always get the same value, but if you re-load it from disk, the calculation will be slightly different from previous calculations. This needs to be fixed. I will make a commit that tags this issue and include a JUnit test file whose test fails because of this problem.

**Comment by schrum2** [comment]
Our troubleshooting seems to reveal that the problem is that the levels are loaded in a different order. So, although all of the rooms are the same for a given dungeon (we believe), looping through them in a different order accumulates small floating point errors.

We're trying to fix this by sorting the rooms according to their structure, but this is proving slightly difficult. In the Level class, we were hoping to use the instance variable stringLevel, but apparently this is null sometimes (always?). Jake said that intLevel already contains the enemies, and would thus be inappropriate to use. There is a method getStringLevel but when I used it (with startingPoint new Point(5, 8)), I got the exception:
```
Exception in thread "main" java.lang.IllegalArgumentException: Invalid GAN code for Zelda: code = -55
	at edu.southwestern.tasks.gvgai.zelda.ZeldaVGLCUtil.convertZeldaTileCodestoGVGAI(ZeldaVGLCUtil.java:204)
```

I guess the sorting will have to be based on the Tile array
